testsuite/tests/singlecluster/authorino/identity/plain/test_jwt_plain_identity.py

```diff
@@ -0,0 +1,91 @@
+"""
+Plain identity test with automation handled by Envoy leveraging the Envoy JWT Authentication filter
+(decoded JWT injected as 'metadata_context' into the response).
+"""
+
+import pytest
+
+from testsuite.httpx.auth import HttpxOidcClientAuth
+from testsuite.gateway.envoy.jwt_plain_identity import JwtEnvoy
+from testsuite.utils import extract_response
+
+
+pytestmark = [pytest.mark.authorino, pytest.mark.standalone_only]
+
+
+@pytest.fixture(scope="module")
+def realm_role(keycloak, blame):
+    """Creates new realm role"""
+    return keycloak.realm.create_realm_role(blame("role"))
+
+
+@pytest.fixture(scope="module")
+def user_with_role(keycloak, realm_role, blame):
+    """Creates new user and adds him into realm_role"""
+    username = blame("someuser")
+    password = blame("password")
+    user = keycloak.realm.create_user(username, password)
+    user.assign_realm_role(realm_role)
+    return user
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, realm_role, blame):
+    """
+    Setup AuthConfig to retrieve identity from given path.
+    Adds rule, that requires user to be part of realm_role to be allowed access.
+    """
+    authorization.identity.add_plain(
+        "plain", "context.metadata_context.filter_metadata.envoy\\.filters\\.http\\.jwt_authn|verified_jwt"
+    )
+    authorization.responses.add_simple("context")
```

Comment:
You might benefit from adding a little bit bigger path here and have fewer keys used with the extract_response function, it will make it more readable I'd say
```suggestion
    authorization.responses.add_simple("context.metadata_context.filter_metadata.envoy.filters")
```